test_runner/performance/test_lfc_prefetch.py

```diff
@@ -0,0 +1,94 @@
+from __future__ import annotations
+
+import random
+import threading
+import time
+from typing import TYPE_CHECKING
+
+import pytest
+from fixtures.log_helper import log
+
+if TYPE_CHECKING:
+    from fixtures.neon_fixtures import NeonEnv
+from fixtures.utils import USE_LFC
+
+
+@pytest.mark.timeout(10000)
+@pytest.mark.parametrize("n_readers", [1, 2, 4, 8])
```

Comment:
20 variants of one test is a lot.

Would it hurt to reduce this to something that exercises the extremes, like:
```
@pytest.mark.parametrize("n_readers", [1, 8])
@pytest.mark.parametrize("n_writers", [0, 1, 8])
```

and/or if the intent of the test is to stress test correctness, then why not just test with the highest numbers?